default-compact.inv

```diff
@@ -0,0 +1,51 @@
+# NOTE: for defining hosts both hostnames and IP addresses forms can be used
```

Comment:
All these lines are commented, so IMHO, this doesn't serve a purpose except as a sample inv file, in which case, the name should be changed